packages/jest-resolve/package.json

```diff
@@ -11,6 +11,7 @@
     "browser-resolve": "^1.11.2",
     "jest-file-exists": "^17.0.0",
     "jest-haste-map": "^18.1.0",
+    "jest-util": "^18.1.0",
```

Comment:
Hey! It's a conscious decision not to make jest-resolve depend on jest-util. Can we move the method you added to be a static method on jest-resolve? I think that makes more sense.